README.md

```diff
@@ -1,108 +1,109 @@
 # Bridge TensorFlow* to run on Intel® nGraph™ backends
 
 This directory contains the code needed to build a TensorFlow 
-plugin that can be used with Intel® nGraph™. nGraph is an [open-source C++ library, compiler and runtime] that provides developers with the means to 
-train and run DNN models efficiently on custom backends: GPU, CPU, and custom 
-silicon.
+plugin to the Intel® nGraph™ Compiler. As part of a TensorFlow 
+toolchain, nGraph can speed up training workloads using CPU; or 
+the nGraph Library and runtime suite can be used to customize 
+and deploy Deep Learning inference models that will "just work" 
+with variety of nGraph-enabled backends: CPU, GPU, and custom 
+silicon.  
 
-## Linux instructions
+*   [Linux instructions](#linux-instructions)
+*   [OS X instructions](#os-x-instructions)
+*   [Debugging](#debugging)
+*   [Support](#support)
+*   [How to Contribute](#how-to-contribute)
 
-There are a few ways you can run nGraph with TensorFlow. They are described in the 
-following section.
 
-## Option 1: Build nGraph bridge from source using existing TensorFlow installation
 
-1. Install TensorFlow using the instructions from the TensorFlow web site
-   https://www.tensorflow.org/install/install_linux
+## Linux instructions
 
-2. Now clone the `ngraph-tf` repo and go to the `ngraph-tf` directory
+You can connect nGraph to an existing TensorFlow installation if 
+you're running TensorFlow v1.11.0-rc1 or greater. For versions 
+released previous to that, we recommend starting with a clean 
+environment and re-building with the instructions in Option 2.
 
-    ```
-    git clone https://github.com/NervanaSystems/ngraph-tf.git
-    cd ngraph-tf
-    git checkout v0.5.0
-    ```
+### Option 1: Use an existing TensorFlow installation
 
-3. Next, build and install nGraph bridge. 
-   :warning: You must be inside the `virtualenv` where TensorFlow installed 
-   during the ngraph-tf build.
+1. If you already have a requisite version resultant from following 
+   the [linux-based install instructions on the TensorFlow website], 
+   you must also instantiate a specific kind of `virtualenv`  to 
+   be able to proceed with the `ngraph-tf` bridge installation. For 
```

Comment:
Um, not sure what is meant by "instantiate a specific kind of `virtualenv`"
Its just a normal virtualenv right?

Do we mean to say something like, "You must instantiate a virtualenv specifically for ngraph-tf..." ? 

Also virtualenv is just the recommended way right? One could also do this on their system python right?
@avijit-nervana

Reply:
The most common way to do virtual envs is with python modules like `python -m venv` for Python 3. This requirement diverges from that norm, so we're calling it out.